packages/roosterjs-editor-api/lib/table/formatTable.ts

```diff
@@ -9,14 +9,14 @@ import { VTable } from 'roosterjs-editor-dom';
  */
 export default function formatTable(
     editor: IEditor,
-    format: Partial<TableFormat>,
+    format: Required<TableFormat>,

```

Comment:
Why change this?

We should keep public function to use Partial so that people can call it with fewer parameters, but change private method to use Required so that we are sure all properties have value